react-with-graphql-tutorial.md

```diff
@@ -46,7 +46,7 @@ Along the way of this React with GraphQL tutorial, you will build a simplified {
 
 {{% chapter_header "GraphQL Query with GitHub's GraphQL API" "graphql-query-github-api" %}}
 
-In this section, you will interact with the GitHub API by only writing queries and mutations without using any React yet. Therefore, you are going to use your GraphiQL application or GitHub's GraphQL Explorer from the previous sections to make your first GraphQL query requests to GitHub's API. Both tools should be authorized to make a request by using your personal access token. Therefore, on the left-hand side of your GraphiQL application, you can fill in GraphQL queries and mutations. Let's add the following query to request data about yourself.
+In this section, you will interact with the GitHub API using queries and mutations without React, so it can use a GraphiQL application or GitHub's GraphQL Explorer to make GraphQL query requests to GitHub's API. Both tools should be authorized to make request using a personal access token. On the left-hand side of your GraphiQL application, you can fill in GraphQL queries and mutations. Add the following query to request data about yourself.
```

Comment:
... so it can use a GraphiQL application or GitHub's GraphQL Explorer to make ... to ... so you can use ...

Reply:
Unclear what you want me to do here.

Reply:
Sorry :) I think it should be more like this: "In this section, you will interact with the GitHub API using queries and mutations without React, so you can use your GraphiQL application or GitHub's GraphQL Explorer to make GraphQL query requests to GitHub's API."